gnomad/gnomadv4_cost_run_extract_rows.py

```diff
@@ -0,0 +1,37 @@
+# LT 1/09/2022
+
+## Cost estimation run on ~5% of partitions
+## annotate gnomADv4 with frequencies and variant QC standard fields
+## export the rows as a hail.Table on a cpg bucket
+
+import hail as hl
+from cpg_utils.hail_batch import output_path
+
+# connect to hail, requester_pays parameters are setup in the dataproc parameters
+hl.init(default_reference='GRCh38')
+
+# gnomAD v4 exomes VDS
+big_vds = hl.vds.read_vds('gs://gnomad/v4.0/raw/exomes/gnomad_v4.0.vds')
+
+# subset to ~5% of partitions (big_vds has 47960 partitions)
+small_vds = hl.vds.VariantDataset(
+    big_vds.reference_data._filter_partitions(range(2500)),
+    big_vds.variant_data._filter_partitions(range(2500)),
+)
```

Comment:
Can you make this controlled by a parameter instead of duplicating the full code?

Reply:
not for a test run intended to be run only once